src/language.rs

```diff
@@ -40,65 +79,55 @@ mod lazy {
 /// These are not of much use right now, and may even be removed from the crate, as there is no
 /// official language specified by the standard except English.
 ///
-/// [Mnemonic]: ../mnemonic/struct.Mnemonic.html
-/// [Seed]: ../seed/struct.Seed.html
+/// [Mnemonic]: ./mnemonic/struct.Mnemonic.html
+/// [Seed]: ./seed/struct.Seed.html
 #[derive(Debug, Clone, Copy)]
 pub enum Language {
-    English
+    English,
+    ChineseSimplified,
+    ChineseTraditional,
+    French,
+    Italian,
+    Japanese,
+    Korean,
+    Spanish,
 }
 
 impl Language {
-    /// Get the [`Language`][Language] value for a specific locale
-    ///
-    /// Not used much at the moment as the standard specifies english
-    ///
-    /// # Example
-    /// ```
-    /// use bip39::{Language};
-    ///
-    /// let lang = Language::for_locale("en_US.UTF-8").unwrap();
-    ///
-    /// ```
-    /// [Language]: ../language/struct.Language.html
-	pub fn for_locale<S>(locale: S) -> Result<Language, Error> where S: Into<String> {
-
-        let l = locale.into();
-
-        let lang = match &*l {
-            "en_US.UTF-8" => Language::English,
-            "en_GB.UTF-8" => Language::English,
-
-            _ => { return Err(ErrorKind::LanguageUnavailable.into()) }
-        };
-
-        Ok(lang)
+    /// Get the word list for this language
+    pub fn wordlist(&self) -> &'static WordList {
+        match *self {
+            Language::English => &lazy::WORDLIST_ENGLISH,
+            Language::ChineseSimplified => &lazy::WORDLIST_CHINESE_SIMPLIFIED,
+            Language::ChineseTraditional => &lazy::WORDLIST_CHINESE_TRADITIONAL,
+            Language::French => &lazy::WORDLIST_FRENCH,
+            Language::Italian => &lazy::WORDLIST_ITALIAN,
+            Language::Japanese => &lazy::WORDLIST_JAPANESE,
+            Language::Korean => &lazy::WORDLIST_KOREAN,
+            Language::Spanish => &lazy::WORDLIST_KOREAN,
```

Comment:
Spanish is pointing to the Korean wordlist (same for the maps)

Reply:
Thanks, corrected!